cmd/sqlflow/main.go

```diff
@@ -51,12 +51,16 @@ func compile(cgName, sqlProgram, datasource string) (string, error) {
 
 func main() {
 	ds := flag.String("datasource", "", "database connect string")
+	logPath := flag.String("log", "", "path/to/log, e.g.: /var/log/sqlflow.log")
 	cgName := flag.String("codegen", "", "SQLFlow compile the input SQL program into Python program using the specified code generator.")
 	flag.StringVar(cgName, "x", "", "short for --codegen")
 	sqlFileName := flag.String("file", "", "execute SQLFlow from file.  e.g. --file '~/iris_dnn.sql'")
 	flag.StringVar(sqlFileName, "f", "", "short for --file")
 	flag.Parse()
 
+	lf := log.New(*logPath)
```

Comment:
What if we want to share a logger across multiple packages, e.g. `pkg/sql` and `cmd/sqlflowserver`? Does it mean we need to pass `lf` around?

Reply:
I need to pass the *pkg/log.Logger* which holds the fields (i.e.: context).